exonum-light-client/src/test/java/com/exonum/client/ExonumHttpClientIntegrationTest.java

```diff
@@ -209,8 +203,7 @@ void getTransactionNotFound() throws InterruptedException {
     // Assert request params
     RecordedRequest recordedRequest = server.takeRequest();
     assertThat(recordedRequest.getMethod(), is("GET"));
-    assertThat(recordedRequest, hasPathStartingWith(TRANSACTIONS));
-    assertThat(recordedRequest.getRequestUrl().queryParameter("hash"), is(id.toString()));
+    assertThat(recordedRequest, hasPath("api/explorer/v1/transactions?hash=" + id.toString()));
```

Comment:
If we use `recordedRequest.getRequestUrl().encodedPath()` in the
matcher, we can make an assertion on the path only (that do not require constructing a full request or comparing a prefix only), and keep separate assertions on the request parameters. Will it make sense?

Reply:
And, importantly, will also make `hasPathStartingWith` obsolete.

Reply:
It's used in `com.exonum.client.ExonumHttpClientWithPrefixIntegrationTest`.

Reply:
I see, well, obsolete for all tests but one :-)